test/ng/directive/ngRepeatSpec.js

```diff
@@ -1141,7 +1143,7 @@ describe('ngRepeat', function() {
       scope.$digest();
       expect($exceptionHandler.errors.shift().message).
           toMatch(
-            /^\[ngRepeat:dupes\] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: {}/);
+            /^\[ngRepeat:dupes] Duplicates in a repeater are not allowed\. Use 'track by' expression to specify unique keys\. Repeater: item in items, Duplicate key: object:3, Duplicate value: /);
```

Comment:
Ah, no, I'm wrong, they're just braces since there's nothing inside and it
was not ESLint but my IDE which was warning incorrectly. I'll restore them.